packages/terra-application-navigation/src/drawer-menu/_DrawerMenu.jsx

```diff
@@ -168,6 +178,7 @@ const DrawerMenu = ({
         ) : null}
         {onSelectHelp ? (
           <DrawerMenuListItem
+            id={utilityItemId(id, "Help")}
```

Comment:
this would generate the same id as if someone used the key of 'help'. Though that is unlikely I'd like to avoid it by giving this a slightly different id:
instead of `${appNavId}-${itemType}-${itemKey}`
something like `${appNavId}-${itemType}${reservedItemKey}` (there might be a better name than `reservedItemKey`)

Same comment for settings and logout.

Reply:
This has been addressed